package.json

```diff
@@ -17,6 +17,8 @@
   "dependencies": {
     "axios": "^0.15.3",
     "body-parser": "^1.15.2",
+    "connect-mongo": "^1.3.2",
```

Comment:
@idandagan1 Ignore these additions for now. I was attempting to fix the session issues.